app/models/game.rb

```diff
@@ -112,18 +119,43 @@ def self.search(title, publisher, tourney, checked, loaned, group)
       result = result.includes(:loans).references(:loans).merge(Loan.where(closed: false, group: group, event: Event.current))
     end
 
-    result.where(culled: false)
+    result.where(status: 0)
   end
 
   def cull_game
     if self.checkouts.where(closed: false).size == 0
-      self.update(culled: true)
+      self.update(status: 1)
       'Game successfully culled!'
     else
       'Game is still checked out! Please return game before culling it.'
     end
   end
 
+  def store_game
+    if self.checked_out?
+      return {
+        error: true,
+        message: 'Game is currently checked out by an attendee and can not be stored!'
+      }
+    end
+
+    if self.status == "storage"
+      self.update(status: 0)
+      return {
+        error: false,
+        message: "Game successfully removed from storage!",
+        removed: true
+      }
+    end
+
+    self.update(status: 2)
```

Comment:
With this line it looks like you would be updating the status of a culled game to stored if you call it on a culled game. Is that intentional?

Reply:
That won't happen. If you try to put in the barcode of a culled game, the get method of game won't find it.